guava/src/com/google/common/math/DoubleMath.java

```diff
@@ -51,6 +53,7 @@
    * This method returns a value y such that rounding y DOWN (towards zero) gives the same result as
    * rounding x according to the specified mode.
    */
+  @SuppressWarnings("expression.unparsable.type.invalid")//Link to issue: https://github.com/typetools/checker-framework/issues/2030( marked as unreproducible)
```

Comment:
If #2030 issue has not been fixed, why is it closed?
Don't state that the issue was marked as unreproducible here. That should be discussed in the issue itself, this comment might get out of date and confuse people.

Reply:
I've reopened the issue again and provided a different test case
@panacekcz

Reply:
Thanks.